.gitignore

```diff
@@ -1,5 +1,36 @@
-/target/
-/bin/
+build/
+.out/
+bin/
+.bin/
+target/
+
+/gradle.properties
+/archive/
+/META-INF
```

Comment:
We need to keep it for MANIFEST.MF

Reply:
It starts with `/`, so it only ignores the `META-INF` in the repository root.

Reply:
Good catch! Ok thanks for clarification